test/api.mocha.coffee

```diff
@@ -350,6 +356,11 @@ describe "API", ->
 
               # Send them invitations
               (_party, cb) ->
+                 # @TODO - _party should be array of newly
```

Comment:
Do the tests put the data into the production database?  I'm guessing not since the database is supposed to have protection against duplicate IDs, but I'm asking just in case. :)